Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -128,12 +128,26 @@ When you start a new project, we provide a default set of configurations with ev
 
 ![](/External/ReadMeImages/MRTK_CopyCreateConfigurationOptions.png)
 
+When profiles are created by the MRTK, they are then placed in the following folder:
+
+> "Assets\MixedRealityToolkit-Generated\CustomProfiles"
+
 At each step in the configuration, you can choose to remove and create a new profile, or simply copy the existing settings and continue to customize:
 
 ![](/External/ReadMeImages/MRTK_CopyProfileOptions.png)
 
-### **[For more information on Configuring the Configuration Profiles](/Documentation/MixedRealityConfigurationGuide.md)**
+### **[For more information on customising the Configuration Profiles](/Documentation/MixedRealityConfigurationGuide.md)**
```

Comment:
we are using "customising" and "customizing" interchangeably... do we want en-GB or en-US spellings?

Reply:
Bugger I and my spellchecker missed ones.  Grrrr